src/TransCreator/utilityMethod.js

```diff
@@ -0,0 +1,23 @@
+import { setNewTransactionAmount } from '../Stats/actionCreator';
```

Comment:
the `utilityMethod.js` filename should be more specific, it should tell what does it do

Reply:
fixed